doc/papers/joss/paper.md

```diff
@@ -7,21 +7,43 @@ tags:
   - spectral elements
   - matrix-free
 authors:
+  - name: Valeria Barra
+    orcid: 0000-0003-1129-2056
+    affiliation: 1
+  - name: Natalie Beams
+    affiliation: 3
   - name: Jed Brown
     orcid: 0000-0002-9945-0639
     affiliation: 1
-  - name: Jeremy Thompson
-    affiliation: 1
-  - name: Valeria Barra
-    orcid: 0000-0003-1129-2056
+  - name: Jean-Sylvain Camier
+    affiliation: 2
+  - name: Veselin Dobrev
+    affiliation: 2
+  - name: Yohann Dudouit
+    affiliation: 2
+  - name:  Leila Ghaffari
     affiliation: 1
-  - name: XXXX TODO Many more authors to be added XXXX
+  - name: Tzanio Kolev
     affiliation: 2
+  - name: David Medina
+    affiliation: 4
+  - name: Thilina Rathnayake
+    affiliation: 5
+  - name: Jeremy Thompson
+    affiliation: 1
+  - name: Stan Tomov
+    affiliation: 3
```

Comment:
@nbeams @camierjs @v-dobrev @YohannDudouit @LeilaGhaffari @tzanio @dmed256 @thilinarmtb @jeremylt @stomov We've tentatively added you to this author list because you are listed as an author on the users manual. Could you please contribute your ORCID (please create an account at https://orcid.org if you don't have one)? (You can leave a comment here and we'll add it, or you can push it in a commit.) By doing so, you're confirming your willingness to be an author pursuant to [JOSS authorship guidelines](https://joss.readthedocs.io/en/latest/submitting.html#authorship). Also, please let us know if anyone has been missed.

Reply:
@abdelfattah83 Cc: #545

Reply:
@pazner Since the plan is to include the Julia interface in the JOSS submission, can you add your ORCID here?